app/services/quotable_service.rb

```diff
@@ -0,0 +1,18 @@
+require 'faraday'
+require 'json'
+
+class QuotableService
+
+  def self.parse(response)
+    JSON.parse(response, symbolize_names: true)
+  end
+
+  def self.conn
+    Faraday.new(url: 'https://api.quotable.io')
+  end
+
+  def self.top_results(query)
+    response = conn.get("/quotes?query=#{query}&limit=10")
```

Comment:
I would look over the docs again. I think the endpoint that more closely gets what we are wanting is `/search/quotes`. Check this out https://github.com/lukePeavey/quotable#search-quotes-beta:~:text=einstein%7Cabraham%2Dlincoln-,Search%20Quotes%20(beta),-GET%20/search/quotes

Reply:
Ahhh you are right I do remeber reading that